rest-api/dao/physical_measurements_dao.py

```diff
@@ -21,7 +21,7 @@
 _COMPOSITION_RESOURCE_TYPE = 'Composition'
 _CREATED_LOC_EXTENSION = 'http://terminology.pmi-ops.org/StructureDefinition/authored-location'
 _FINALIZED_LOC_EXTENSION = 'http://terminology.pmi-ops.org/StructureDefinition/finalized-location'
-_PM_SYSTEM = 'http://terminology.pmi-ops.org/CodeSystem/physical-measurements'
```

Comment:
In the changes I am working on right now, we are switching to `http://terminology.pmi-ops.org/CodeSystem/physical-measurements` as the code system.

Reply:
OK. Are you doing that across the board? (We already have data in the system using physical-evaluation.)